### PR TITLE
chore(core): Update traefik version to v2.2

### PR DESCRIPTION
**Pull request by netoax**
What does this implement/fix? Explain your changes.
---------------------------------------------------
In order to route AMQP messages we need to use a new traefik mechanism
that enables routing generic TCP messages. For that reason, this patch
migrates the traefik reverse proxy to its new version.

https://docs.traefik.io/migration/v1-to-v2/

Does this close any currently open issues?
------------------------------------------
Nops

Any relevant logs, error output, etc?
-------------------------------------
Nops.

Any other comments?
-------------------
Nops.

### CODE REVIEW

**Review by lcbm — APPROVED** [review]
Awesome patch! :confetti_ball:

File: stacks/core/dev/stage-1.yml [hunk 1]
```diff
@@ -68,8 +72,9 @@ services:
       replicas: 1
       labels:
         - traefik.enable=true
-        - traefik.frontend.rule=HostRegexp:bt,bt.{domain:[a-zA-Z0-9.]+}
-        - traefik.port=8080
+        - traefik.http.services.babeltower.loadbalancer.server.port=8080
+        - traefik.http.routers.babeltower.rule=HostRegexp(`{subdomain:api}.{domain:[a-zA-Z0-9.]+}`)
```
Comment (lcbm) [inline stacks/core/dev/stage-1.yml:76]: Please update the [docs](https://github.com/CESARBR/knot-cloud/tree/master/stacks/core#configure-dns) to reflect these changes :smile:

**Review by lcbm — CHANGES_REQUESTED** [review]

File: stacks/core/dev/stage-1.yml (hunk 1 above)
Comment (netoax) [reply stacks/core/dev/stage-1.yml:76]: Done.

**Review by lcbm — APPROVED** [review]
LGTM :confetti_ball:

**Pull request merged**